### ISSUE TITLE
Docker Installation Failed

### ISSUE THREAD

**Issue by manuelvogel12** [ADVRHumanoids/concert_description#28]
While Installing concert_description by running, 
`./build-docker.bash`,
the following Error Occurred:
```
calling "cmake /home/user/concert_ws/src/centauro_cartesio/. -DCMAKE_INSTALL_PREFIX=/home/user/concert_ws/install -DCMAKE_BUILD_TYPE=RelWithDebInfo"
returned 1
..[centauro_cartesio] configuring failed
[concert_all] failed to install dependency centauro_cartesio
(failed)
The command '/bin/bash -ic forest grow concert_all --verbose --jobs 4 --pwd user' returned a non-zero code: 1
```

Happens with the newest version, even after re-cloning the repository

**Comment by EdoardoRomiti** [comment]
This is fixed in [v1.0.19](https://github.com/ADVRHumanoids/concert_description/releases/tag/v1.0.19). @manuelvogel12 can you confirm?

**Comment by manuelvogel12** [comment]
Yes, everything works perfectly again, thanks
